tilt_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: tilt_joint
    joint_speed:8.0
    motor:
        id: 13
        init: 2275
        min: 0
        max: 4095
